dotbot/cli.py

```diff
@@ -28,12 +37,24 @@ def main():
         parser = ArgumentParser()
         add_options(parser)
         options = parser.parse_args()
-        if (options.super_quiet):
+        if options.super_quiet:
             log.set_level(Level.WARNING)
-        if (options.quiet):
+        if options.quiet:
             log.set_level(Level.INFO)
-        if (options.verbose):
+        if options.verbose:
             log.set_level(Level.DEBUG)
+        plugin_directories = list(options.plugin_dirs)
+        if not options.disable_built_in_plugins:
+          plugin_directories.append(os.path.join(os.path.dirname(__file__), '..', 'plugins'))
+        plugin_paths = []
+        for directory in plugin_directories:
+          for plugin_path in glob.glob(os.path.join(directory, '*.py')):
+            plugin_paths.append(plugin_path)
+        for plugin_path in options.plugins:
+            plugin_paths.append(plugin_path)
+        for plugin_path in plugin_paths:
+            abspath = os.path.abspath(plugin_path)
+            module.load(plugin_path)
```

Comment:
`abspath` isn't used here. Bug? Or does this work fine, and we can get rid of line 56?